app/server/src/main/scala/org/bitcoins/server/BitcoindRpcBackendUtil.scala

```diff
@@ -0,0 +1,99 @@
+package org.bitcoins.server
+
+import org.bitcoins.core.api.chain.ChainQueryApi
+import org.bitcoins.core.api.chain.ChainQueryApi.FilterResponse
+import org.bitcoins.core.api.node.NodeApi
+import org.bitcoins.core.protocol.BlockStamp
+import org.bitcoins.core.protocol.transaction.Transaction
+import org.bitcoins.core.util.{BitcoinSLogger, FutureUtil}
+import org.bitcoins.crypto.{DoubleSha256Digest, DoubleSha256DigestBE}
+import org.bitcoins.rpc.client.common.BitcoindRpcClient
+import org.bitcoins.wallet.Wallet
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/** Useful utilities to use in the wallet project for syncing things against bitcoind */
+object BitcoindRpcBackendUtil extends BitcoinSLogger {
```

Comment:
Can we just move this into the `bitcoindRpc` project so we can share this between here and https://github.com/bitcoin-s/bitcoin-s/blob/fa388c744847081788a98352c304a377e815904c/testkit/src/main/scala/org/bitcoins/testkit/chain/SyncUtil.scala#L51

Reply:
Then the bitcoind rpc project will need to depend on wallet